cdap-ui/app/directives/epoch/charts.js

```diff
@@ -72,7 +72,7 @@ ngEpoch.controller('epochController', function ($scope, $compile, caskWindowMana
         }
         else if (newVal && newVal.length) {
           for (var i = 0; i < newVal.length; i++) {
-            $scope.me.push([newVal[i]]);
+            $scope.me.push(newVal[i]);
```

Comment:
Any specific reason why this is changing? This will cause javascript error to be thrown right?